packages/date/src/index.js

```diff
@@ -357,57 +361,90 @@ export function format( dateFormat, dateValue = new Date() ) {
 }
 
 /**
- * Formats a date (like `date()` in PHP), in the site's timezone.
+ * Formats a date (like `date()` in PHP).
  *
- * @param {string}                           dateFormat PHP-style formatting string.
- *                                                      See php.net/date.
- * @param {(Date|string|Moment|null)}        dateValue  Date object or string,
- *                                                      parsable by moment.js.
+ * @param {string}                  dateFormat PHP-style formatting string.
+ *                                             See php.net/date.
+ * @param {Date|string|Moment|null} dateValue  Date object or string, parsable
+ *                                             by moment.js.
+ * @param {string|number|null}      timezone   Timezone to output result in or a
+ *                                             UTC offset. Defaults to timezone from
+ *                                             site.
  *
- * @return {string} Formatted date.
+ * @see {@link https://en.wikipedia.org/wiki/List_of_tz_database_time_zones|Timezones}
+ * @see {@link https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC|UTC Offsets}
```

Comment:
I'm not sure one way or the other if this is valid formatting for `@see`, but at the very least, the documentation generator doesn't handle it.

```suggestion
 * @see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 * @see https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC
```

You'd need to rerun the documentation generator locally as well (happens on pre-commit), not just commit these changes.

Reply:
Fixed in 4cafd10.